apps/src/sites/studio/pages/init/loadSpritelab.js

```diff
@@ -4,12 +4,36 @@ import SpriteLab from '@cdo/apps/p5lab/spritelab/SpriteLab';
 import blocks from '@cdo/apps/p5lab/spritelab/blocks';
 import skins from '@cdo/apps/p5lab/skins';
 import levels from '@cdo/apps/p5lab/levels';
+import {getDefaultListMetadata} from '@cdo/apps/assetManagement/animationLibraryApi';
```

Comment:
I believe we are going to need to add all of these changes to `loadPoetry.js` as well.